lib/internal/test_runner/mock.js

```diff
@@ -157,17 +161,10 @@ class MockTracker {
         'options.setter', setter, "cannot be used with 'options.getter'"
       );
     }
+    validateField(object, methodName);
```

Comment:
This is what the code now on line 169 is also doing. The idea is to look up the function and then assert that it is a function. I think this can be dropped.

Reply:
this is different. This makes sure that the property exists in the object instance. At 169 it makes sure it's a function. Without it throws a "Cannot convert null to an object" error.

Reply:
nevermind. Just dropped it!